installer/manifests/keptn/values.yaml

```diff
@@ -365,7 +365,7 @@ lighthouseService:
       cpu: "200m"
 
 statisticsService:
-  enabled: false
+  enabled: true
```

Comment:
Does it make sense to have it enabled by default? Or was this an accidental change :D

Reply:
Actually, having it set to fals was an accidental change, which caused us to discover the problem with nginx :D :dagger:

Reply:
but I agree, we should think about whether false should be the default, actually....
@thisthat ?